src/fields/fq.rs

```diff
@@ -642,10 +649,23 @@ impl FieldExt for Fq {
     ]);
 
     fn sqrt_ratio(num: &Self, div: &Self) -> (Choice, Self) {
+        #[cfg(not(feature = "alloc"))]
+        {
+            let _ = (num, div);
+            unimplemented!()
+        }
```

Comment:
Perhaps make an issue in this repo for resolving this.

Reply:
Now that the entire `FieldExt` trait is behind the `std` feature flag, this is irrelevant.